scenarios/kind_modelservice_inference-sim.sh

```diff
@@ -15,12 +14,12 @@ export LLMDBENCH_VLLM_MODELSERVICE_DECODE_CPU_NR=0
 export LLMDBENCH_VLLM_MODELSERVICE_PREFILL_CPU_NR=0
 export LLMDBENCH_VLLM_MODELSERVICE_DECODE_CPU_MEM=100Mi
 export LLMDBENCH_VLLM_MODELSERVICE_PREFILL_CPU_MEM=100Mi
-export LLMDBENCH_VLLM_MODELSERVICE_URI="hf://facebook/opt-125m"
+export LLMDBENCH_VLLM_MODELSERVICE_URI_PROTOCOL="hf"
 export LLMDBENCH_DEPLOY_MODEL_LIST="facebook/opt-125m:facebook/opt-125m"
-export LLMDBENCH_VLLM_COMMON_PVC_MODEL_CACHE_SIZE=3Gi
+#export LLMDBENCH_VLLM_COMMON_PVC_MODEL_CACHE_SIZE=3Gi
 export LLMDBENCH_HARNESS_PVC_SIZE=3Gi
-export LLMDBENCH_VLLM_COMMON_PVC_NAME=model-pvc
-export LLMDBENCH_CONTROL_DEPLOY_IS_OPENSHIFT=0
-export LLMDBENCH_CONTROL_RESOURCE_LIST=deployment,httproute,service,gateway,gatewayparameters,inferencepool,inferencemodel,cm,ing,pod,job
+#export LLMDBENCH_VLLM_COMMON_PVC_NAME=model-pvc
+#export LLMDBENCH_CONTROL_DEPLOY_IS_OPENSHIFT=0
+#export LLMDBENCH_CONTROL_RESOURCE_LIST=deployment,httproute,service,gateway,gatewayparameters,inferencepool,inferencemodel,cm,ing,pod,job
```

Comment:
I think this can be deleted given the fix in `setup/env.sh`

Reply:
Commented out lines removed